### DISCUSSION TITLE
CFG Scale times two - Is this the new normal ?

### DISCUSSION THREAD

**Discussion by swamp-cabbage** [vladmandic/sdnext#1716] [Q&A] [answered]
Apologies in advance if my search missed the correct answer.

I am using "Original Recipe" pipeline, 0f81cfc2 build from today.

Using "From Image" I have two CFG scale sliders - In addition to the "Image CFG Scale" slider.

![isthisthenewnormal](https://github.com/vladmandic/automatic/assets/133680212/d24c3211-21ca-4cde-9c38-f750ef243f7c)

Is this a new feature that I missed, or is it an issue ? 

Thanks !

**Comment by vladmandic** [comment, marked as answer]
renamed second one, that was a typo.

**Reply by gsgoldma** [reply]
is there any documentation on what guidance rescale does?

**Reply by vladmandic** [reply]
For algorithm that helps with over/underexpose.
Implemented in diffusers, no effect in original backend.

**Comment by swamp-cabbage** [comment]
> renamed second one, that was a typo.

Thank you Sir !
